dist/angular-leaflet-directive.js

```diff
@@ -3018,6 +3027,10 @@ angular.module("leaflet-directive")
   };
 });
 
+angular.module("leaflet-directive").service('leafletLogger', ["nemSimpleLogger", function(nemSimpleLogger) {
+  return nemSimpleLogger.spawn();
```

Comment:
Create our own instance of an independent logger where leaflet-directive's logging domain is separate of everything else. "leafletLogger" is then used everywhere else where $log used to be.